File: Assets/Php_htdocs/Retrieve/Aggregate.cs
using Newtonsoft.Json;
using System.Collections;
using System.Collections.Generic;
using Unity.VisualScripting;
using UnityEngine;
using UnityEngine.Networking;

public class Aggregate : MonoBehaviour
{
    [SerializeField] Transform _summaryPanelContainer;
    [SerializeField] GameObject _summaryPanelPrefab;

    struct WrapperJSONResult
    {
        public string SortingColValue;
        public string CharacterCounts;
        public int MaxAtk;
        public int MaxDef;
        public int MaxHP;
        public string MaxAtkName;
        public string MaxDefName;
        public string MaxHPName;
        public float TotalRevenue;
    }
    private string _columnName;    

    void Start()
    {
        CallAggregation("region");   
    }

    public void CallAggregation(string columnName)
    {
        this._columnName = columnName;
        this.StartCoroutine(DoAggregation());
    }

    IEnumerator DoAggregation()
    {
        WWWForm form = new WWWForm();
        form.AddField("FIELD_SortingColumn", _columnName);

        //using (UnityWebRequest handler = UnityWebRequest.Post("http://localhost/Retrieve/Aggregate.php", form))
        using (UnityWebRequest handler = UnityWebRequest.Post(ConnectionSettings.SERVER_ADDRESS + "/Retrieve/Aggregate.php", form))
        {
            yield return handler.SendWebRequest();
            EraseCurrSummaryPanels();


            if(handler.error == null)
            {
                Debug.Log(handler.downloadHandler.text);
                UseResults(JsonConvert.DeserializeObject<WrapperJSONResult[]>(handler.downloadHandler.text));
            }
            else
            {
                Debug.LogError($"Aggregation failed {handler.error}");
            }
        }
    }

    void EraseCurrSummaryPanels()
    {
        foreach(Transform child in _summaryPanelContainer.transform)
        {
            Destroy(child.gameObject);
        }
    }


    void UseResults(WrapperJSONResult[] results)
    {
        foreach(WrapperJSONResult result in results)
        {
            string imgPath = this._columnName switch
            {
                "region" => "Regions/" + result.SortingColValue + "_Emblem_Night",
                "weapon_type" => "Weapons/UI_GachaTypeIcon_" + result.SortingColValue,
                "vision" => "Elements/UI_Buff_Element_" + result.SortingColValue,
                "model" => "Models/" + result.SortingColValue,
                "rarity" => "Rarity/Rarity_" + result.SortingColValue,
                _ => "Unknown"
            };

            string revenue = (result.TotalRevenue == 0) ? "NO DATA" : "$" + result.TotalRevenue.ToString("#,#");

            GameObject newPanel = Instantiate(_summaryPanelPrefab, _summaryPanelContainer);
            newPanel.GetComponent<SummaryPanel>().SetValues(
                result.SortingColValue,
                result.MaxAtk.ToString("#,#") + '\n' + result.MaxAtkName,
                result.MaxDef.ToString("#,#") + '\n' + result.MaxDefName,
                result.MaxHP.ToString("#,#") + '\n' + result.MaxHPName,
                result.CharacterCounts,
                revenue,
                imgPath
                );
        }
    }
}

File: Assets/Php_htdocs/Retrieve/Aggregate.php
<?php
require("../Connect.php");

$FIELD_SortingColumn = $_POST['FIELD_SortingColumn'];

$sql = "SELECT (".$FIELD_SortingColumn.") AS sortingColValue, 
	COUNT('".$FIELD_SortingColumn."') AS `counts`, 
	MAX(atk_90_90) AS `max_atk`,
	MAX(def_90_90) AS `max_def`,
	MAX(hp_90_90) AS `max_hp`
FROM table_masterlist GROUP BY (".$FIELD_SortingColumn.");";

$query = mysqli_query($CONNECTION, $sql) or die("[1] Aggregation failed");


$OUT_Results = array();
while($result = mysqli_fetch_assoc($query)){
    $newRes = new stdClass();    

    $newRes->SortingColValue = $result['sortingColValue'];
    $newRes->CharacterCounts = $result['counts'];
    $newRes->MaxAtk = $result['max_atk'];
    $newRes->MaxDef = $result['max_def'];
    $newRes->MaxHP = $result['max_hp'];


    $atkQuery = mysqli_query($CONNECTION, "SELECT character_name FROM table_masterlist WHERE atk_90_90 = '".$newRes->MaxAtk."' AND (".$FIELD_SortingColumn.") = '".$newRes->SortingColValue."';") or die("[2]: Failed to query max atk name");
    $defQuery = mysqli_query($CONNECTION, "SELECT character_name FROM table_masterlist WHERE def_90_90 = '".$newRes->MaxDef."' AND (".$FIELD_SortingColumn.") = '".$newRes->SortingColValue."';") or die("[3]: Failed to query max def name");
    $hpQuery = mysqli_query($CONNECTION, "SELECT character_name FROM table_masterlist WHERE hp_90_90 = '".$newRes->MaxHP."' AND (".$FIELD_SortingColumn.") = '".$newRes->SortingColValue."';") or die("[4]: Failed to query max hp name");

    if(mysqli_num_rows($atkQuery) < 1) {die("[5] Failed to find max attacker name");}
    if(mysqli_num_rows($defQuery) < 1) {die("[6] Failed to find max def name");}
    if(mysqli_num_rows($hpQuery) < 1) {die("[7] Failed to find max hp name");}
    $newRes->MaxAtkName = mysqli_fetch_assoc($atkQuery)['character_name'];
    $newRes->MaxDefName = mysqli_fetch_assoc($defQuery)['character_name'];
    $newRes->MaxHPName = mysqli_fetch_assoc($hpQuery)['character_name'];

    array_push($OUT_Results, $newRes);
}
echo json_encode($OUT_Results);
?>